backend/src/main/kotlin/cz/loono/backend/schedule/TaskScheduler.kt

```diff
@@ -0,0 +1,16 @@
+package cz.loono.backend.schedule
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+
+@Component("taskScheduler")
+class TaskScheduler @Autowired constructor(
```

Comment:
Make use of Spring's Component annotation? I.e.
```suggestion
class TaskScheduler(
```